src/components/Text/index.tsx

```diff
@@ -19,22 +19,21 @@ export const textVariants = Object.keys(typography) as TextVariant[]
 
 /**
  * Generate all styles available for text based on prominence and variants
- * @param prominence
- * @param variant
- * @param theme
  */
 const generateStyles = ({
   prominence,
   color,
   variant,
   theme,
   oneLine,
+  disabled,
```

Comment:
A disabled text ?? 🤔

Reply:
Sometime parent (for instance Button or a more complex component) can be disabled, so text must have related style, it purely a display/style prop.